packages/ui-library/src/components/slider-two-values/range-legend-min-max-slider/index.test.ts

```diff
@@ -3,50 +3,47 @@ import '@boiler/ui-library/dist';
-    decrementIcon: 'blrMinus',
-    onChange: () => null,
+  theme: 'Light',
+  rangeInputId: 'range-legend-cmpt',
+  startValue: '200$',
+  endValue: '400$',
+  size: 'lg',
+  list: ['100$', '200$', '300$', '400$', '500$'],
+  stepFactor: 1,
+  btnVariant: 'primary',
+  incrementIcon: 'blrPlus',
+  decrementIcon: 'blrMinus',
+  onChange: () => null,
 };
 
 describe('blr-range-legend-min-max-slider', () => {
-    it('is having two sliders containing the right className', async () => {
-        const element = await fixture(BlrRangeLegendMinMaxSliderRenderFunction(sampleParams));
-        const inputWrapper = querySelectorDeep('.range-wrapper', element.getRootNode() as HTMLElement);
-        const textareas = querySelectorAllDeep('input', inputWrapper?.getRootNode() as HTMLElement);
-        const classNameInputOne = textareas[0]?.className;
-        const classNameInputTwo = textareas[1]?.className;
-        
-       expect(classNameInputOne).to.contain('range'); 
-       expect(classNameInputTwo).to.contain('range'); 
-    });
+  it('is having two sliders containing the right className', async () => {
+    const element = await fixture(BlrRangeLegendMinMaxSliderRenderFunction(sampleParams));
+    const inputWrapper = querySelectorDeep('.range-wrapper', element.getRootNode() as HTMLElement);
+    const textareas = querySelectorAllDeep('input', inputWrapper?.getRootNode() as HTMLElement);
+    const classNameInputOne = textareas[0]?.className;
+    const classNameInputTwo = textareas[1]?.className;
+
+    expect(classNameInputOne).to.contain('range');
+    expect(classNameInputTwo).to.contain('range');
+  });
 
-    it('properly renders legend', async () => {
-        const element = await fixture(BlrRangeLegendMinMaxSliderRenderFunction(sampleParams));
-        const inputWrapper = querySelectorDeep('.range-wrapper', element.getRootNode() as HTMLElement);
-        const rangeNumbersElement = querySelectorDeep('.range__numbers', inputWrapper?.getRootNode() as HTMLElement);
-        const allRangePoints = querySelectorAllDeep('.range__point', rangeNumbersElement?.getRootNode() as HTMLParagraphElement);
+  it('properly renders legend', async () => {
+    const element = await fixture(BlrRangeLegendMinMaxSliderRenderFunction(sampleParams));
+    const inputWrapper = querySelectorDeep('.range-wrapper', element.getRootNode() as HTMLElement);
+    const rangeNumbersElement = querySelectorDeep('.range__numbers', inputWrapper?.getRootNode() as HTMLElement);
+    const allRangePoints = querySelectorAllDeep(
+      '.range__point',
+      rangeNumbersElement?.getRootNode() as HTMLParagraphElement
+    );
 
-        allRangePoints.forEach(point => {
-            expect(point).to.exist;
-        });
-        expect(allRangePoints.length).to.equal(sampleParams.list.length)
+    allRangePoints.forEach((point) => {
+      expect(point).to.exist;
     });
-})
+    expect(allRangePoints.length).to.equal(sampleParams.list.length);
+  });
+});
```

Comment:
The formatting changes seem out of scope for this PR and could be removed.
What do you think? I'm curious, especially since i'm new to the team.
Should we just fix these things on the fly whenever we encounter them?

Reply:
To be honest, I have no glue. I think this came when I solved the merge conflict  from develop. I don't touch this file in this way.